gensrc/thrift/MVMaintenance.thrift

```diff
@@ -0,0 +1,74 @@
+// This file is made available under Elastic License 2.0
```

Comment:
I have a question. Why you use Thrift not Protobuf?

Reply:
Because existed QueryPlan is serialized as thrift, the major part of Job Description if query plan, I don't want to invent another serialization format.